Makefile

```diff
@@ -5,6 +5,10 @@ APP_DIR := ./app
 GO := GO111MODULE=on go
 GOBIN := $(shell go env GOPATH)/bin
 
+KIND_API_IP ?= $(shell make -sC _run/kube kind-k8s-ip)
+KIND_HTTP_PORT ?= $(shell make -sC _run/kube app-http-port)
+KIND_VARS ?= KIND_APP_PORT="$(KIND_HTTP_PORT)" KIND_K8S_IP="$(KIND_API_IP)"
```

Comment:
why create new var names for the same thing?  can you use this?

```
KIND_VARS ?= KIND_HTTP_PORT="$(KIND_HTTP_PORT)" KIND_API_IP="$(KIND_API_IP)"
```

Reply:
Originally I saw a Makefile error of something like "cannot assign variable name recursively", however possibly after adding the quote interpolation it ceased. So good catch, cleaning up the name spam, and making consistently `KIND_APP....`